mappings/net/minecraft/client/gui/hud/debug/AbstractDebugChartRenderer.mapping

```diff
@@ -0,0 +1,47 @@
+CLASS net/minecraft/unmapped/C_bkzzefxr net/minecraft/client/gui/hud/debug/AbstractDebugChartRenderer
+	FIELD f_dehsgucw CHART_HEIGHT I
+	FIELD f_kumhtbzz TEXT_COLOR I
+		COMMENT The color of the text used in the chart. {@code e0e0e0} as a hex color code.
+	FIELD f_roauydkh LINE_WIDTH I
+	FIELD f_wmaqprzm sampleLog Lnet/minecraft/unmapped/C_fnzofwlw;
+	METHOD <init> (Lnet/minecraft/unmapped/C_mavozmpp;Lnet/minecraft/unmapped/C_fnzofwlw;)V
+		ARG 2 sampleLog
+	METHOD m_ctbaldbu lineLength (D)I
+		ARG 1 value
+	METHOD m_julaomxm appendUnit (D)Ljava/lang/String;
+		ARG 1 value
+	METHOD m_phxaaunk getWidth (I)I
+		COMMENT Returns the width of the debug chart: {@code width} bounded by a maximum width.
+		ARG 1 width
+	METHOD m_rjcqkpru drawLabel (Lnet/minecraft/unmapped/C_sedilmty;Ljava/lang/String;II)V
+		ARG 2 label
+		ARG 3 x
+		ARG 4 y
+	METHOD m_sdtiatjq renderChart (Lnet/minecraft/unmapped/C_sedilmty;II)V
+		ARG 2 x
+		ARG 3 width
+	METHOD m_srllwucw getSampleColor (DDIDIDI)I
+		COMMENT Gets the color for the sample's line representing its performance impact.
+		COMMENT @return The interpolated color based on the {@code delta}'s relation to the low, medium, and high color deltas.
+		ARG 1 delta
+			COMMENT the point between the {@code lowColorDelta} and {@code highColorDelta} to interpolate the color of
+		ARG 3 lowColorDelta
+			COMMENT the {@code delta} value below which the interpolated color should be exactly the {@code lowColor}
+		ARG 5 lowColor
+			COMMENT the color representing a low sample value
+		ARG 6 mediumColorDelta
+			COMMENT the {@code delta} value for which the interpolated color should be exactly the {@code mediumColor}
+		ARG 8 mediumColor
+			COMMENT the color representing a medium sample value
+		ARG 9 highColorDelta
+			COMMENT the {@code delta} above which the interpolated color should be exactly the {@code highColor}
+		ARG 11 highColor
+			COMMENT the color representing a high sample value
```

Comment:
```suggestion
		COMMENT @return The interpolated color based on the {@code delta}'s relation to the minimum, medium, and maximum color deltas.
		ARG 1 delta
			COMMENT the point between the {@code minColorDelta} and {@code maxColorDelta} to interpolate the color of
		ARG 3 minColorDelta
			COMMENT the {@code delta} value below which the interpolated color should be exactly the {@code minColor}
		ARG 5 minColor
			COMMENT the color representing a minimum sample value
		ARG 6 mediumColorDelta
			COMMENT the {@code delta} value for which the interpolated color should be exactly the {@code mediumColor}
		ARG 8 mediumColor
			COMMENT the color representing a medium sample value
		ARG 9 maxColorDelta
			COMMENT the {@code delta} above which the interpolated color should be exactly the {@code maxColor}
		ARG 11 maxColor
			COMMENT the color representing a maximum sample value
```
They're bounded, so min/max seems more appropriate.
Though maybe just the deltas should be min/max and the colors should remain low/high? I'm not sure.

Reply:
I initially had it like this, but I thought that low and high was more understandable with their meaning... not set in stone on that though.  They should definitely match though, the deltas and colors.

Reply:
I kind of see what you mean (I think), but the javadocs are clearly describing upper and lower bounds.
I also think there's no need to have `Color` in each of the delta names:
```suggestion
		COMMENT @return the interpolated color based on the {@code delta}'s relation to the minimum, medium, and maximum color deltas.
		ARG 1 delta
			COMMENT the point between the {@code minDelta} and {@code maxDelta} to interpolate the color of
		ARG 3 minDelta
			COMMENT the {@code delta} value below which the interpolated color should be exactly the {@code minColor}
		ARG 5 minColor
			COMMENT the color representing a minimum sample value
		ARG 6 mediumDelta
			COMMENT the {@code delta} value for which the interpolated color should be exactly the {@code mediumColor}
		ARG 8 mediumColor
			COMMENT the color representing a medium sample value
		ARG 9 maxDelta
			COMMENT the {@code delta} above which the interpolated color should be exactly the {@code maxColor}
		ARG 11 maxColor
			COMMENT the color representing a maximum sample value
```

Reply:
To me, it's clearer that they are the points for the color provided... but maybe that's not clearer for other people?

Reply:
Hm, I suppose that makes sense.
I'm also reconsidering the min/max part. It's arguably referencing implementation details to call them min/max.
I think your original names are best.

Reply:
I mean, I definitely think it's a big grey area.  They are minimums and maximums, they are cutoffs... but at the same time, the values don't cut off, the line lengths don't cut off, it's only the colors.  I think my names are possibly better at avoiding any confusing connotations with respect to that.

Not that it matters much, they are descriptive enough names that likely will hardly ever be seen by anyone other than us lol, I doubt debug charts are particularly popular focuses for mods!